pheweb/load/phenolist.py

```diff
@@ -20,69 +20,51 @@ def run():
     code2pheno = {pheno['phenocode']:pheno for pheno in phenolist}
     code2pheno_prev = {pheno['phenocode']:pheno for pheno in phenolist_prev}
     tag2name = {}
-    tag2codes = od()
-    
+    code2tag = {}
+    tag2index = {}
+
     with open(sys.argv[3]) as f:
         f.readline()
         for line in f:
             s = line.strip().split('\t')
-            tag2codes[s[0].strip()] = {}
             tag2name[s[0].strip()] = s[len(s)-1].strip()
 
     with open(sys.argv[4]) as f:
+        f.readline()
+        for line in f:
+            s = line.strip().split('\t')
+            code2tag[s[2]] = s[0]
+            if s[0] not in tag2index:
+                tag2index[s[0]] = len(tag2index)
+
+    with open(sys.argv[5]) as f:
```

Comment:
again not needed for now but in the future for readability/flexiblity/consistency, it would be good for all CLI tools to have argparse and hence named arguments with meaningful names instead of indexes

Reply:
Yup this far these are one-off changing for every release